packages/nx/src/native/plugins/js/ts_import_locators.rs

```diff
@@ -589,6 +602,55 @@ mod find_imports {
     use assert_fs::TempDir;
     use swc_common::comments::NoopComments;
 
+    #[test]
+    fn should_not_include_ignored_imports() {
+        let temp_dir = TempDir::new().unwrap();
+        temp_dir
+            .child("test.ts")
+            .write_str(
+                r#"
+  // nx-ignore-next-line
+  import 'a';
+
+/* nx-ignore-next-line */
+import 'a1';
+
+/*      nx-ignore-next-line */
+import 'a2';
+
+/**
+  * nx-ignore-next-line
+  */
+import 'a3';
+
+/*
+  nx-ignore-next-line
+  */
+import 'a4'; import 'a5';
+
+/* prettier-ignore */ /* nx-ignore-next-line */
+import 'a4'; import 'a5';
+
+/* nx-ignore-next-line */ /* prettier-ignore */
+import 'a4'; import 'a5';
+
```

Comment:
Adding this as a test case fails :frown:
```
// nx-ignore-next-line 
// prettier-ignore
import 'a6';
```

Reply:
That makes sense to me because the import is not on the next line following the comment.

Switching the order of the comments will work.

This doesn't work using the typescript code either.